earth_enterprise/src/installer/install_fusion.sh

```diff
@@ -418,6 +418,9 @@ parse_arguments()
 				if [ $IS_NEWINSTALL == false ]; then
 					echo -e "\nYou cannot modify the fusion user name using the installer because Fusion is already installed on this server."
 					parse_arguments_retval=1
+					# Don't show the User Group dialog since it is invalid to change the fusion
+					# username once fusion is installed on the server
+					show_user_group_recommendation=false
```

Comment:
We should disable this a few lines down as well where the user has previously installed fusion and is trying to change the group.

Reply:
I would update copyright notice here from `Copyright 2017 Google Inc.` to `Copyright 2017 Google Inc, 2019 Open GEE Contributors`

Reply:
bcd994d Updates the code for group.